From 25b004fe0881d01a094416588d9260feefb71af1 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 04 Jul 2018 15:59:36 +0300
Subject: [PATCH] board: habana: fix asic pdu ip address

The pdu used for both, the onix and the asic boards is the same.
Do not split it in the code.
Note, we might require the splitting some time later.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/ip_pdu b/board/habanalabs/common/ip_pdu
index 8169e3f..ae351c6 100755
--- a/board/habanalabs/common/ip_pdu
+++ b/board/habanalabs/common/ip_pdu
@@ -9,13 +9,13 @@
 fi
 
 pdu_cred="--user=snmp --password=1234"
+pdu_uri=http://192.168.10.80
+
 case "$board" in
 *asic*)
-	pdu_uri=http://192.168.10.120
 	pwr_id=1
 	;;
 *onix)
-	pdu_uri=http://192.168.10.80
 	pwr_id=1
 	;;
 *)
